google-cloud-firestore/src/main/java/com/google/cloud/firestore/QueryDocumentSnapshot.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.firestore;
+
+import com.google.common.base.Preconditions;
+import com.google.firestore.v1beta1.Document;
+import com.google.firestore.v1beta1.Value;
+import com.google.protobuf.Timestamp;
+import java.util.Map;
+import javax.annotation.Nonnull;
+import org.threeten.bp.Instant;
+
+/**
+ * A QueryDocumentSnapshot contains data read from a document in your Firestore database as part of
```

Comment:
s/in your Firestore database/in a Firestore database/

Reply:
Done